src/backend/server.ts

```diff
@@ -210,11 +226,10 @@ const logger = {
   server.route(terms);
   server.route(privacy);
```

Comment:
We don't need the about, terms, or privacy routes here since they're already at the top.

Reply:
nice catch 👌🙏